src/linux/linux-host-session.vala

```diff
@@ -419,7 +408,47 @@ namespace Frida {
 			return result;
 		}
 
-		public async uint spawn (string package_name, string? activity_name) throws Error {
+		public async uint spawn (string program, HostSpawnOptions options) throws Error {
+

```

Comment:
Blank line should be dropped for consistency.

Reply:
ok